tests/conftest.py

```diff
@@ -3245,22 +3245,13 @@ def sqlite_connection_string() -> str:
     return f"sqlite:///{db_file_path}"
 
 
-@pytest.fixture
-def fds_data_context_datasource_name() -> str:
-    return "sqlite_datasource"
-
-
 @pytest.fixture
 def fds_data_context(
-    sa,
-    fds_data_context_datasource_name: str,
-    empty_data_context: AbstractDataContext,
-    sqlite_connection_string: str,
+    sa, empty_data_context: AbstractDataContext, sqlite_connection_string: str
```

Comment:
nit: You might want to return the same of the things added to the context so tests don't need to know the names. It does look like knowing the names in the tests is done a lot though.